spec/support/factory_bot.rb

```diff
@@ -7,13 +7,12 @@
     password_confirmation { 'password' }
   end
 
-    factory :park do
-      sequence(:name) {|n| "#{n}" }
-      sequence(:address) {|n| "#{n}" }
-      sequence(:city) {|n| "#{n}" }
-      sequence(:state) {|n| "#{n}" }
-      sequence(:zip) {|n| "#{n}" }
-      sequence(:description) {|n| "#{n}" }
-
-    end
+  factory :park do
+    sequence(:name) {|name| name }
```

Comment:
This is actually the same as the above -- `n` is just now a different variable :) I was thinking 

```suggestion
    sequence(:name) {|n| "Park #{n}" }
    sequence(:address) {|n| "#{n} Main Street" }
    city: { "Boston" }
```

(maybe just using sequencing when it makes sense, too)

Let me know if you'd like to chat through this in person!